src/components/FwbTextarea/FwbTextarea.vue

```diff
@@ -7,6 +7,13 @@
         :rows="rows"
         :class="textareaClasses"
         :placeholder="placeholder"
+        :form="form"
+        :disabled="disabled"
+        :maxlength="maxlength"
+        :minlength="minlength"
+        :required="required"
+        :readonly="readonly"
+        :wrap="wrap"
```

Comment:
We can achieve this with `inheritAttrs: false` and v-bind="$attrs":
```vue
<template>
    ...
    <textarea 
        v-bind="$attrs"
        ...
    />
    ....
</template>

<script>
    ...
    defineOptions({
        inheritAttrs: false,
    })
    ...
</script>
```

this would greatly reduce the number of props needed, increased readability, and be more future prof (we would support new attributes without any code changes)

docs: https://vuejs.org/guide/components/attrs.html#disabling-attribute-inheritance

Reply:
Sure, this would save a lot of work and keep the code simple. Let me look into it